manifests/init.pp

```diff
@@ -4,6 +4,7 @@
 #
 class openvpn (
   String $openvpn_dir,
+  String $openvpn_path,
```

Comment:
Should this be an `Absolutepath` or whatever its called from stdlib instead of a string?